### ISSUE TITLE
new term samp_source_mat_cat [MIXS:0001154] has two different definitions

### ISSUE THREAD

**Issue by only1chunts** [GenomicsStandardsConsortium/mixs#210]
**Current term details**
In the NewFoodPackages tab in the MIXS6 spreadsheet the term samp_source_mat_cat [MIXS:0001154] has been given different definitions and expected values for two of the individual packages:
```
Term name - sample source material category
Term ID - MIXS:0001154
Structured comment name - samp_source_mat_cat
Value syntax - {termLabel} {[termID]}
```
**Packages - food-human foods, food-animal and animal feed, & food-farm environment**
Definition - This is the scientific role or category that the subject organism or material has with respect to an investigation.  This field accepts terms listed under specimen source material category (http://purl.obolibrary.org/obo/GENEPIO_0001237).
Expected value - GENEPIO:0001237
Example - environmental (swab or sampling) [GENEPIO:0001732]

**Packages - food-food production facility**
Definition - This is the scientific role or category that the subject organism or material has with respect to an investigation.  This field accepts terms listed under specimen (http://purl.obolibrary.org/obo/OBI_0100051).	
Expected value - OBI:0100051
Example - environmental swab specimen [OBI:0002613]

**Suggested update(s)**
We need there to be a single definition to work for all packages. please can someone create that?

**Comment by only1chunts** [comment]
This new term is included in #201

**Comment by lschriml** [comment]
Fixed: (http://purl.obolibrary.org/obo/GENEPIO_0001237 or http://purl.obolibrary.org/obo/OBI_0100051).
